README.md

```diff
@@ -34,14 +34,28 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic HTML is writing HTML code with tags that indicate meaning to their content. It is possible to write 
+a perfectly functional and nice looking website just using a bunch of div elements, however this is confusing
+to search engines and fellow developers. 
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+Display block is the default for css. It means that items will take up the entire width of the web page and 
```

Comment:
there are different elements that default to different styling, a span is inline, a div is block